File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/GetSearchUsersExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class GetSearchUsersExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            var response = new UsersApi(config).GetSearchUsers(
                projectKey: "projectKey_string",
                environmentKey: "environmentKey_string"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling UsersApi#GetSearchUsers: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/GetSearchUsersExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.UsersApi(configuration).getSearchUsers(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  undefined, // q
  undefined, // limit
  undefined, // offset
  undefined, // after
  undefined, // sort
  undefined, // searchAfter
  undefined, // filter
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling UsersApi#getSearchUsers:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
